docs/containers/app-service.md

```diff
@@ -88,6 +89,24 @@ In the previous section, the image is pushed to a remote container registry. Now
 
     ![Web Application](images/app-service/webapp-homepage.png)
 
+## Deploy to ACI
+
+You can easily run your containers in Azure using [Azure Container Instances (ACI)](https://azure.microsoft.com/services/container-instances/). First, deploy the container to a container registry, such as Docker Hub or ACR, as described earlier in this article. Then, find the container in the **Registries** section of the Docker view. Use the **Refresh** button if you don't see it there. Right-click the entry for the image you want, and choose **Deploy image to Azure Container Instances**.
+
+![Deploy to ACI menu item](images/app-service/deploy-aci-menu.png)
+
+Choose an existing context, or use **Create new ACI Context**, and then choose the resource group. The instance is set up and started in Azure. 
+
+The context you created is added to the **Contexts** pane in the Docker view. When you change the active context, it affects the images shown in the **Containers** pane as well as the context used in the terminal.
+
+![ACI in contexts pane](images/app-service/deploy-aci-contexts.png)
+
+ When the image finishes the start-up process and becomes available, an entry for the image appears in the **Containers** pane. You can then manage the container instance by right-clicking on the entry. To monitor the logs, choose **View Logs**.
+
```

Comment:
How about this for the last sentence...
For example, to monitor the logs choose **View Logs**.